plugin.json

```diff
@@ -30,6 +30,12 @@
         "type": "bool",
         "default": false
       },
+      {
+        "key": "ProcessPostsFromBots",
+        "display_name": "Process Posts from Bots:",
+        "type": "bool",
+        "help_text": "When enabled, the plugin will be able to edit posts made by bots."
+      },
```

Comment:
Should this be a per-link setting?

Reply:
I agree this could be a per link setting. But grasping for uses case examples....